src/Sylius/Bundle/ChannelBundle/Doctrine/ORM/ChannelRepository.php

```diff
@@ -24,28 +22,8 @@ class ChannelRepository extends EntityRepository implements ChannelRepositoryInt
     /**
      * {@inheritdoc}
      */
-    public function findMatchingHostname($hostname)
-    {
-        $queryBuilder = $this->getQueryBuilder();
-
-        $queryBuilder
-            ->andWhere($queryBuilder->expr()->like('o.url', ':hostname'))
-            ->setParameter('hostname', '%'.$hostname.'%')
-        ;
-
-        return $queryBuilder->getQuery()->getOneOrNullResult();
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function findDefault()
```

Comment:
Let's discuss this today, sounds reasonable, but can lead to some unexpected behavior, especially when poorly documented.